app/experimenter/legacy-ui/rapid/components/visualization/PrimaryMetricTable.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+
+import { displaySelectOptionList } from "experimenter-rapid/components/experiments/utils";
+import { featureOptions } from "experimenter-rapid/components/forms/ExperimentFormOptions";
+import {
+  PRIMARY_METRIC_COLUMNS,
+  TABLE_LABEL,
+  DISPLAY_TYPE,
+} from "experimenter-rapid/components/visualization/constants/analysis";
+import ResultsRow from "experimenter-rapid/components/visualization/ResultsRow";
+import { ExperimentData } from "experimenter-types/experiment";
+
+const getStatistics = (
+  feature: string,
+): Array<{
+  name: string;
+  displayType: DISPLAY_TYPE;
+  branchComparison?: string;
+}> => {
+  const featureMetricID = `${feature}_ever_used`;
+
+  // Make a copy of `PRIMARY_METRIC_COLUMNS` since we modify it.
+  const primaryMetricStatisticsList = PRIMARY_METRIC_COLUMNS.map(
+    (statistic) => {
+      statistic["value"] = featureMetricID;
+      return statistic;
+    },
+  );
+
+  return primaryMetricStatisticsList;
+};
+
+const PrimaryMetrics: React.FC<{
+  experimentData: ExperimentData;
+  feature: string;
+}> = ({ experimentData, feature }) => {
+  const featureDisplayName = displaySelectOptionList(
+    featureOptions,
+    feature,
+  ).pop();
+  const statsData = experimentData.analysis?.overall || {};
+  const primaryMetricStatistics = getStatistics(feature);
+  const metricKey = `${feature}_ever_used`;
+
+  return (
+    <div>
+      <h2 className="font-weight-bold">{featureDisplayName}</h2>
+      <table className="table text-right h5 mb-5">
+        <thead>
+          <tr>
+            <th scope="col"></th>
+            {PRIMARY_METRIC_COLUMNS.map((value) => {
+              return (
```

Comment:
(Optional) You can return straight JSX if you'd like `...value) => (<th...`, same with the map below